.github/actions/install-all-deps/action.yml

```diff
@@ -54,6 +54,12 @@ runs:
       run: |
         python -m pip install --upgrade virtualenv
         python -m virtualenv venv
+    - name: Install uv
+      shell: bash
+      run: |
+        . ${{ env.VENV_ACTIVATE }}
+        python -m pip install uv
```

Comment:
@abidlabs `uv` is installed here with pip. bBut we should use the standalone installer instead:

```
curl -LsSf https://astral.sh/uv/install.sh | sh
```